### DISCUSSION TITLE
Unable to Output Complete 2D Visualization Results for Steady Wind in FAST.Farm Simulation

### DISCUSSION THREAD

**Discussion by Steven647** [OpenFAST/openfast#2535] [Q&A] [answered]
Hello Everyone，

I'm a Ph.D. student studying wind turbine aerodynamics, and I need some assistance with FAST.Farm.I am using **FAST.Farm** to simulate the wake of a wind turbine. The inflow wind is defined as steady wind in the InflowWind module, setting the . However, the wake visualization results **cannot** output content beyond the 2D(2 Diameter). 

I am using the NREL 5MW wind turbine model with a rotor diameter of 126 m. In the InflowWind module, the parameter **WindType** is set to 1, and the wind speed is 11.4 m/s. In the main input file of FAST.Farm, the parameter **Mod_Wake** is set to 1, corresponding to the **Polar** Wake. In the VISUALIZATION section, **NOutDisWindYZ** is set to 9, with the corresponding **OutDisWindX** specifying 9 values. In the OUTPUT section, **NOutDist** is set to 8, with the corresponding **OutDist** defining 8 values at 1D intervals downstream of the turbine. However, in the visualization results of the wake, there is no wake data beyond a downstream distance of 2 rotor diameters from the turbine. The results are as follows:

OXY Plane
![OXY](https://github.com/user-attachments/assets/3e508569-a906-4bc2-a536-fff5e8828fd1)

OXZ Plane
![OXZ](https://github.com/user-attachments/assets/84a72a0d-5ab2-40ea-994a-7ba79c0b9420)

The main input file of FAST.Farm is as follows:
[FAST.Farm.txt](https://github.com/user-attachments/files/17903289/FAST.Farm.txt)

I would appreciate any suggestions or insights on resolving this issue！

Best regards,
Steven Liu

**Comment by jjonkman** [comment, marked as answer]
Dear @Steven647

Your `DT_Low` is quite small for 11.4 m/s, so, the wake planes will be quite closely spaced and you would need many more wake planes to extend the wake further downstream.  But it would be better to increase `DT_Low` to a more reasonable value.

I would suggest following the FAST.Farm modeling guidance when defining the spatial-temporal resolution: https://openfast.readthedocs.io/en/main/source/user/fast.farm/ModelGuidance.html.

Best regards,

**Reply by Steven647** [reply]
Dear @jjonkman 

Thank you very much for your response and the suggestions provided. Following your advice, I have obtained the desired wake results. Once again, thank you!

Best regards,
Steven Liu
